src/main/java/io/jenkins/plugins/coverage/targets/CoverageResult.java

```diff
@@ -77,11 +76,7 @@
     private String name;
     private String tag;
 
-    /**
-     * Properties to store a change request coverage diff with the latest successful build from the target branch
-     */
-    private float changeRequestCoverageDiffWithTargetBranch = 0;
-    private String linkToBuildThatWasUsedForComparison = null;
+    private String referenceBuildUrl = null;
```

Comment:
Renaming fields (and public getters) normally is not a good thing:
- it breaks serialization (if no migration is provided)
- it breaks callers (or subclasses)

However, since the serialization uses Java serialization (https://github.com/jenkinsci/code-coverage-api-plugin/blob/master/src/main/java/io/jenkins/plugins/coverage/CoverageProcessor.java#L691) and not the XStream serialization previous results will be broken anyway if we add new content. So it might be ok here. 

@oleg-nenashev @cizezsy what was the rationale behind that decision not to use Jenkins default XML serialization in this plugin?